packages/palette/src/elements/MultiSelect/MultiSelect.tsx

```diff
@@ -184,71 +179,60 @@ const Options = styled(Box)`
   -webkit-overflow-scrolling: touch;
 `
 
-const STATES = {
-  default: css`
-    color: ${themeGet("colors.black60")};
-    border-color: ${themeGet("colors.black30")};
-  `,
-  complete: css`
-    color: ${themeGet("colors.black100")};
-    border-color: ${themeGet("colors.black60")};
-  `,
-  focus: css`
-    color: ${themeGet("colors.black100")};
-    border-color: ${themeGet("colors.black60")};
-  `,
-  hover: css`
-    color: ${themeGet("colors.black100")};
-    border-color: ${themeGet("colors.black60")};
-  `,
-  disabled: css`
-    color: ${themeGet("colors.black30")};
-    border-color: ${themeGet("colors.black30")};
-  `,
-  error: css`
-    color: ${themeGet("colors.black100")};
-    border-color: ${themeGet("colors.red100")};
-  `,
-}
-
 type ContainerProps = Pick<
   MultiSelectProps,
-  "complete" | "disabled" | "error" | "hover" | "focus"
+  "complete" | "disabled" | "error" | "hover" | "focus" | "visible"
 >
 
 const Container = styled(Clickable)<ContainerProps>`
   position: relative;
   width: 100%;
-  height: 50px;
-  border: 0;
-  border-bottom: 1px solid;
   /* 24px = space.1 + 4px-wide caret + space.1 */
-  padding: 0 24px 0 ${themeGet("space.1")};
-  transition: background-color 0.25s, border-color 0.25s;
+  padding: 16px 24px 16px ${themeGet("space.1")};
+  font-family: ${themeGet("fonts.sans")};
+  border: 1px solid;
+  border-radius: 3px;
+  border-color: ${themeGet("colors.black30")};
+  cursor: pointer;
+  line-height: 1;
+  transition: color 0.25s, background-color 0.25s, border-color 0.25s;
+  background-color: ${themeGet("colors.white100")};
 
   ${(props) => {
     return css`
-      ${STATES.default}
-      ${props.complete && STATES.complete}
-      ${props.hover && STATES.hover}
-      ${props.focus && STATES.focus}
-      ${props.disabled && STATES.disabled}
-      ${props.error && STATES.error}
-
+      ${MULTISELECT_STATES.default}
+      ${props.complete && MULTISELECT_STATES.completed}
+      ${props.hover && MULTISELECT_STATES.hover}
+      ${props.focus && MULTISELECT_STATES.focus}
+      ${props.disabled && MULTISELECT_STATES.disabled}
+      ${props.error && MULTISELECT_STATES.error}
+    
       &:hover {
-        ${STATES.hover}
+        ${MULTISELECT_STATES.hover}
       }
 
-      &:focus {
-        ${STATES.focus}
-      }
+      ${props.visible && MULTISELECT_STATES.focus}
 
       &:disabled {
         cursor: default;
-        ${STATES.disabled}
+        ${MULTISELECT_STATES.disabled}
       }
     `
   }}
 
   ${caretMixin}
 `
+
+const StyledLabel = styled.label`
+  position: absolute;
+  top: 50%;
+  left: 5px;
+  padding: 0 5px;
+  pointer-events: none;
+  transform: translateY(-50%);
+  transition: 0.25s cubic-bezier(0.64, 0.05, 0.36, 1);
```

Comment:
I realize these transitions are copy-pasted from somewhere. Can we please get some kinda extracted constant for these? cc @rajsam003

Reply:
I can support on extracting this to a const @MrSltun but makes sense to include in a follow up PR

Reply:
I agree, I'll extract it from all the components into a shared transition in a separate PR!

Reply:
hey, @dzucconi @MrSltun is there a specific place in palette where theses shared constants are supposed to go?

Reply:
I suppose I'd make a file in the same directory as this [shadows file](https://github.com/artsy/palette/blob/main/packages/palette/src/helpers/shadow.ts) — transitions.ts or something?